akka-cluster/src/main/scala/akka/cluster/ClusterEvent.scala

```diff
@@ -451,7 +451,7 @@ object ClusterEvent {
       val removedMembers = oldGossip.members diff newGossip.members
       val removedEvents = removedMembers.map(m ⇒ MemberRemoved(m.copy(status = Removed), m.status))
 
-      (new VectorBuilder[MemberEvent]() ++= memberEvents ++= removedEvents).result()
+      (new VectorBuilder[MemberEvent]() ++= removedEvents ++= memberEvents).result()
```

Comment:
Why this change?

Reply:
MemberRemoved must be published before MemberUp, e.g. when restarted
in other DC.